python/numeric-python.ipynb

```diff
@@ -961,7 +981,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.5"
+   "version": "3.10.10"
```

Comment:
We should stay with 3.11.5, or newer versions.

Reply:
will look into that tomorrow